hack/cr-lokistack.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  logStore:
+    type: lokistack
+    lokistack:
+      name: lokistack-dev
+  collection:
+    logs:
+      type: fluentd
```

Comment:
I wonder if we should have `vector` in our example for LokiStack. :slightly_smiling_face: